src/dhcp_fsm.erl

```diff
@@ -198,7 +198,7 @@ bound(Pkg = #dhcp_package{xid = _XId, message_type = request},
             case delegate(request, Pkg, State) of
                 {ok, RPkg, State1} ->
                     Timeout = dhcp_package:get_option(ip_address_lease_time, RPkg),
```

Comment:
Line 200 is too long:                     Timeout = dhcp_package:get_option(ip_address_lease_time, RPkg),.